Feature/i enumerable i enumerator

BI003 opened the pull request.

BI003 committed:
В папке “Infrastructure” создан проект “BankSystem.Data”. В новом проекте создана папка “Storages”, в нее добавлен класс “ClientStorage”, представляющий
хранилище клиентов.

BI003 committed:
В хранилище добавлена коллекция List<Client> для
хранение клиентов банка.
Реализован публичный метод добавления
новых клиентов.
В классе тестов проверена работа хранилища, добавление клиентов в
хранилище и  выборки из нее.
● найти самого молодого клиента;
● найти самого старого клиента;
● вычислить средний возраст клиентов;

BI003 committed:
Была реализована идентичная работа, но на примере сотрудников банка.

PetrovskiAlex commented on BancSystem.App.Tests/BancSystem.App.Tests.csproj:
```diff
@@ -17,6 +17,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\BankSystem.App\BankSystem.App.csproj" />
+    <ProjectReference Include="..\BankSystem.Data\BankSystem.Data.csproj" />
```
предполагался другой проект тестов

PetrovskiAlex commented on BancSystem.App.Tests/ClientStorageTest.cs:
```diff
@@ -0,0 +1,91 @@
+
+            foreach (var client in clients)
+            {
+                storage.AddClient(client);
```
стоило добавить метод AddRange в хранилище который принимает массив клиентов

BI003 committed:
Добавил метод "AddRange" в классы "ClientStorage" и "EmployeeStorage", чтобы упростить добавление нескольких клиентов.
В тестах заменил цикл foreach на вызов нового метода AddRange.

BI003 committed:
Был создан новый проект тестов в папке "Infarastructure".
В него были перемещены тестовые классы "ClientStorageTest" и "EmployeeStorageTest"
